include/tbl/cli_cmds.h

```diff
@@ -99,10 +99,14 @@ CLI_CMD(VCL_STATE,
 
 CLI_CMD(VCL_DISCARD,
 	"vcl.discard",
-	"vcl.discard <configname|label>",
-	"Unload the named configuration (when possible).",
-	"",
-	1, 1
+	"vcl.discard <configname|label>...",
+	"Unload the named configurations (when possible).",
```

Comment:
unrelated to the patch: could "when possible" be read as "maybe delayed"? Would "if possible" be better?

Reply:
It doesn't show up in the line range you selected but "when possible" was already present, and it is sometimes "not possible" to discard a VCL (it is active or a dependency of others).

Reply:
yes, I know, which is why I wrote _unrelated to the patch_

Reply:
Yes, I know, but I can't read...

Reply:
("if possible" +1)